roles/openshift_node/tasks/main.yml

```diff
@@ -53,13 +55,13 @@
     - regex: '^CONFIG_FILE='
       line: "CONFIG_FILE={{ openshift_node_config_file }}"
   notify:
-  - restart openshift-node
+  - restart {{ product_type }}-node
```

Comment:
The handler name is `restart node`. Replace `restart {{ product_type }}-node` with `restart node` there should be 2 instances of this.